audio-player.js

```diff
@@ -84,40 +84,40 @@ module.exports = class AudioPlayer {
       throw new Error(
         'Some form of audio playback capability is required');
     }
+    if (this.canUseAudio) {
```

Comment:
This check is unnecessary. It can never be true due to the if statement on line 83.